package.json

```diff
@@ -3,7 +3,11 @@
   "description": "Hoodie Client Core",
   "main": "index.js",
   "scripts": {
-    "test": "standard && npm run -s test:node | tap-spec",
+    "build": "npm-run-all -p build:*",
```

Comment:
Just wondering, why did you decide to go with this setup instead of what we do in the other repos with "build" and "postbuild" tasks? Shall we adapt the others?